docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,68 @@
+
+The feature is also used implicitly by ASP.NET applications: the set of packages composing the ASP.NET Web framework is installed as part of the setup packages authored by Microsoft. When publishing an ASP.NET application, a special task in the SDK finds the right manifest and configures the publication task to use it, resulting in the published application being trimmed to only include the application's packages, and not the framework's packages.
+
+## Publishing an application against a target manifest
+
+If you have a target manifest file on disk, you can specify it when publishing your app with the [`dotnet publish`](../tools/dotnet-publish.md) command:
+
+```
+dotnet publish --manifest path/to/the/target-manifest.csproj
+```
+
+The resulting published application should only be deployed to an environment that has the packages described in the target manifest. Failing to do so would result in the application not starting.
+
+It's possible to specify multiple target manifests when publishing an application. The application will then be trimmed for the union of packages specified in those target manifests.
+
+## Specifying a target manifest in the project file
+
+Instead of specifying a target manifest in a [`dotnet publish`](../tools/dotnet-publish.md) command, it's possible to specify the manifest or manifests to use in the project file under a `TargetManifest` tag.
+
+```xml
+<PropertyGroup>
+  <TargetManifest>path/to/the/target-manifest.csproj</TargetManifest>
+</PropertyGroup>
+```
+
+This should only be done when the target environment for the application is well-known. This is the case for example for ASP.NET projects, and for that reason, ASP.NET project templates include a target manifest (with a special value that is mapped to the corresponding target manifest's path when publishing):
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetManifest>aspnet/2.0.0</TargetManifest>
+  </PropertyGroup>
+```
+
+A different case would be an open-source project: the users of the project will likely deploy to a variety of different production environments, which may have different sets of packages pre-installed. Maintainers of the project can't make assumptions about the target manifest, and users should instead rely on the `--manifest` option of [`dotnet publish`](../tools/dotnet-publish.md) instead.
+
+## Preparing a runtime environment
+
+The administrator of a runtime environment can optimize for certain types of applications by building a runtime package store and the corresponding target manifest.
+
+The first step is to create an empty project and to add the packages that must compose the runtime package store. The runtime package store can then be provisioned by running a `dotnet store --manifest [target-manifest.csproj]` command. Multiple target manifest paths can be passed to the [`dotnet store`](../tools/dotnet-store.md) command.
```

Comment:
Do we want to have a snippet here of what one of these `target-manifest.csproj` files looks like?